frontend/src/page/OptionsSider/ActiveSliderSelect.tsx

```diff
@@ -1,68 +1,89 @@
-
 import React, {useState} from "react";
-import {TextField} from "@mui/material";
-import { useSearchParams} from "react-router-dom";
-
+import {FormControl, InputLabel, MenuItem, Select, SelectChangeEvent, TextField} from "@mui/material";
+import {useSearchParams} from "react-router-dom";
 
 
 export function ActiveSliderSelect() {
 
     const [searchParams, setSearchParams] = useSearchParams();
     let tmpStartAddress = Number(searchParams.get('startaddresse'));
     let tmpQuantity = Number(searchParams.get('faderquantity'));
+    let tmpUniverse = Number(searchParams.get('universe'));
+    const [universe, setUniverse] = React.useState(tmpUniverse);
     const [address, setAddress] = useState(tmpStartAddress);
     const [quantity, setQuantity] = useState(tmpQuantity);
 
     const handleOnChangeAddress = (event: React.ChangeEvent<HTMLInputElement>) => {
-        let tmpAddress = Number(event.target.value )
-        if( tmpAddress < 1 ){
+        let tmpAddress = Number(event.target.value)
```

Comment:
Same here would add constants like MIN_ADRESS_VALUE for the 1,
and MAX_ADRESS_QUANTITY_SUM or so?